client/signup/steps/site-or-domain/index.jsx

```diff
@@ -64,56 +64,83 @@ class SiteOrDomain extends Component {
 			// translators: %s is a domain name
 			buyADomainTitle = translate( 'Just buy %s', { args: [ domainName ] } );
 		}
+		const buyADomainDescription = translate( 'Add a site later.' );
+
+		const newSiteTitle = translate( 'New site' );
+		const newSiteDescription = translate(
+			'Customize and launch your site. Free domain for the first year on annual plans.',
+			{
+				comment: 'Description for new site option, periods added for screen reader pause',
+			}
+		);
+		const newSiteVisualDescription = translate(
+			'Customize and launch your site.{{br/}}{{strong}}Free domain for the first year on annual plans.{{/strong}}',
+			{
+				components: {
+					strong: <strong />,
+					br: <br />,
```

Comment:
These line-breaks are are read as "empty group" by VoiceOver and catch focus when navigating through the page.

![Markup on 2025-06-10 at 15:47:20](https://github.com/user-attachments/assets/4f90baa7-d9fa-4950-995c-aab51e3c91de)

I think we could consider hiding them, e.g.:

```suggestion
					br: <br aria-hidden="true" />,
```

Reply:
Added in https://github.com/Automattic/wp-calypso/pull/104033/commits/b1dc04e86a7458c3a6d5447e89ea2d10d6dab3c0